src/test/java/test/dependent/BaseOrderMethodTest.java

```diff
@@ -22,7 +22,7 @@ public void a_second0() {
 
   @Test(
       groups = {"3"},
-      dependsOnGroups = {"2.0"})
+      dependsOnGroups = {"2.0", "2.1"})
```

Comment:
I think this was a case where the dependency was being assumed by the test, but wasn't explicit.